strax/processing/peak_building.py

```diff
@@ -104,8 +104,8 @@ def find_peaks(hits, adc_to_pe,
 
 
 @export
-@numba.jit(nopython=True, nogil=True, cache=True)
-def store_downsampled_waveform(p, wv_buffer):
+@numba.jit(nopython=True, nogil=True, cache=False)
+def store_downsampled_waveform(p, wv_buffer, wv_top_buffer, wv_bottom_buffer):
```

Comment:
Could, you please also update the doc string? Something like "same as ... but for ..." would be enough.

Reply:
Check